blog/19-05-2022-announcing-open-feature.md

```diff
@@ -0,0 +1,48 @@
+---
+slug: "openfeature-a-standard-for-feature-flagging"
+title: "OpenFeature - a standard for feature flagging"
+date: 2022-05-19
```

Comment:
Did you back date this internally? If so, no problem! I just want to make sure before it's merged.